lib/ref-transform.js

```diff
@@ -36,7 +36,9 @@ module.exports = class SetTransform {
            {{${originalName}}} should be {{${typos[originalName]}}}
         `);
       }
-      if (!originalName.includes('ref')) {
+      if (!originalName || 
```

Comment:
could you mention template itself for this case?

Reply:
Sorry don't understand what you mean

Reply:
sorry, I mean, could you add template example with `originalName` = false | undefined or such?

Reply:
no clue how to reproduce, this was in a real world app with hundreds of templates, and the error didn't allow me to see which template was causing it.